### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by nem035**
[Readme Preview](https://github.com/enkidevs/seapig/blob/14a17214ffdb616aef0b02e04cef52d6595dd582/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,7 +261,266 @@ class Form extends Component {
...
+```
+
+Ok simple enough. Now let's consider a few potential updates we might need to make this component.
+
+Let's imagine a requirement comes in that each page has to render a custom menu item that is unique to it, maybe some icon designated for that portion of our site that may or may not be an anchor as well.
+
+Sure, not a problem, instead of us worrying about element types and icons for each page, let's just allow them to pass that in. Of course we don't want to render blank items if nothing is provided:
+
+```jsx
+const Header = ({ order, PageIcon }) => (
```
Comment (stefanoeb) [inline README.md]: `order` is not being used in the component

File: README.md [hunk 2]
```diff
@@ -242,6 +538,6 @@ const {
 })
 ```
 
-## License
+## <a name="licence">License</a>
```
Comment (stefanoeb) [inline README.md]: typo licence

File: README.md [hunk 3]
```diff
@@ -224,7 +261,266 @@ class Form extends Component {
...
+{/* or */}
+<Header PageIcon={<a href="/settings"><FontAwesome name="cog" /></a>} />
+```
+
+Alright, things are getting a bit messy but still manageable.
+
+Headers usually show user info, assuming the user is authenticated.
+
+```jsx
+const Header = ({ order, PageIcon, authenticated }) => (
```
Comment (stefanoeb) [inline README.md]: same here: order prop not used

**Comment by nem035** [comment]
@stefanoeb yeah, I was trying to keep it interesting/funny but I'm not too happy with the result. 
 Writing is hard 😓 . Did you mean those expressions make it too informal? Or perhaps say "simple" when something isn't simple?

**Comment by lizTheDeveloper** [comment]
They make it seem simple when it might not be simple (simple for you != simple always). Docs don’t have to be formal or informal by convention, and friendly docs are better. The way to friendly-up some text is to crack some jokes or be encouraging, rather than sounding conversational.

**Comment by stefanoeb** [comment]
Yeah, writing is hard indeed 😄 

1. Exactly, 'simple' and 'easy' are relative to the person knowledge. Therefore may not be the case for programming/react newbies.
2. Less is more, getting rid of meaningless sentences will get the example more objective and dry.

File: README.md [hunk 4]
```diff
@@ -1,29 +1,63 @@
+<p align="center">
+	<a href="https://www.npmjs.org/package/seapig">
+  	<img src="https://media.giphy.com/media/X2C5xcQLdVGda/giphy.gif" width="300" height="300" alt="seapig">
+  </a>
+</p>
+
 # seapig 🌊🐷
 
-![Sea Pig](https://media.giphy.com/media/X2C5xcQLdVGda/giphy.gif)
+[Utility for generalized composition of React components](https://www.npmjs.org/package/seapig)
```
Comment (lizTheDeveloper) [inline README.md]: Leave the gif! Add the acronym from down below up here.

File: README.md [hunk 5]
```diff
@@ -1,29 +1,63 @@
...
+Using this pattern in React allows us to have the rendering control with the [minimal api surface area](https://www.youtube.com/watch?v=4anAwXYqLG8) without the need for configuration props for all state and data variations that our component can have.
+
+This is where compound components come to the rescue.
+
+A compound component encloses the state and behavior for a group of components but still gives the rendering control for its variable parts back to the external user. When using it, you only need to worry about the actual parts that are different.
+
+A concrete example of a compound component on the Web today is the `<select>` element. It allows us to externally specify it's rendering structure with `<option>` and `<optiongroup>` elements but hides away all the complexity of rendering those in a certain way or handling/delegating events.
+
+*If you're interested in more examples of compound components, [Ryan Florence](https://twitter.com/ryanflorence) has a [great talk](https://www.youtube.com/watch?v=hEGg-3pIHlE) on this topic.*
+
```
Comment (lizTheDeveloper) [inline README.md]: Use a block quote >

> this is a block quote

**Pull request merged**